knowledge-base/cannot-implicitly-convert-type-telerik.reporting.xxxreportsource-to-telerik.reportviewer.html5.webforms.reportsource-error.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Cannot implicitly convert type 'Telerik.Reporting.XXXReportSource' to 'Telerik.ReportViewer.Html5.WebForms.ReportSource' error when using HTML5 WebForms Report Viewer
+description: Cannot implicitly convert type 'Telerik.Reporting.XXXReportSource' to 'Telerik.ReportViewer.Html5.WebForms.ReportSource' error when using HTML5 WebForms Report Viewer.
+type: how-to
+page_title: Cannot implicitly convert type 'Telerik.Reporting.XXXReportSource' to 'Telerik.ReportViewer.Html5.WebForms.ReportSource' error when using HTML5 WebForms Report Viewer
+slug: cannot-implicitly-convert-type-telerik.reporting.xxxreportsource-to-telerik.reportviewer.html5.webforms.reportsource-error
+res_type: kb
+---
+
+## Description
+
+Cannot implicitly convert type **'Telerik.Reporting.XXXReportSource'** to **'Telerik.ReportViewer.Html5.WebForms.ReportSource'** error when setting the ReportSource of the HTML5 WebForms Report Viewer.  
+  
+
```

Comment:
Are there steps to reproduce the problem? Also, add the exact error in "Error Message" section.

Reply:
Add Cause/Possible Cause section with more info on when and why the error appears. For example in what scenarios and which cases.